tests/server.rs

```diff
@@ -391,6 +396,22 @@ pub fn http_1_1() {
     assert!(&read.starts_with(b"HTTP/1.1 500 Internal Server Error\r\n"), "{:?}", httpbis::misc::BsDebug(&read));
 }
 
+#[cfg(unix)]
+#[test]
+pub fn http_1_1_unix() {
+    env_logger::init().ok();
+
+    let _server = ServerTest::new_unix("/tmp/rust_http2_test".to_owned());
```

Comment:
Fixed path in `/tmp` is not ideal. Something like [that](https://github.com/rust-lang-nursery/tempdir) could be used.